pindel/pindel_cat.cwl

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+
+class: Workflow
+label: "Per-interval pindel"
+inputs:
+    reference:
+        type: File
+        secondaryFiles: [".fai"]
+    tumor_bam:
+        type: File
+        secondaryFiles: .bai
+    normal_bam:
+        type: File
+        secondaryFiles: .bai
+    interval_list:
+        type: File
+    insert_size:
+        type: int
+        default: 400
+outputs:
+    per_interval_pindel_out:
+        type: File
+        outputSource: cat/pindel_out
+steps:
+    sed:
```

Comment:
I'd suggest an awk instead of sed.  The interval start coordinate needs to be converted to zero-based instead of one-based coordinates to make a real BED file.